vm/integer_test.go

```diff
@@ -22,32 +22,39 @@ func TestIntegerClassSuperclass(t *testing.T) {
 	}
 }
 
-func TestIntegerArithmeticOperation(t *testing.T) {
+func TestIntegerArithmeticOperationWithInteger(t *testing.T) {
 	tests := []struct {
 		input    string
 		expected interface{}
 	}{
-		{`1 + 2`, 3},
-		{`10 + 0`, 10},
-		{`22 - 10`, 12},
-		{`2 - 10`, -8},
-		{`5 * 20`, 100},
-		{`4 % 2`, 0},
-		{`10 % 3`, 1},
-		{`6 % 4`, 2},
-		{`25 / 5`, 5},
-		{`25 > 5`, true},
-		{`4 > 6`, false},
-		{`-5 < -4`, true},
-		{`100 < 81`, false},
-		{`5 ** 4`, 625},
-		{`25 / 5`, 5},
-		{`1 / 1 + 1`, 2},
-		{`0 / (1 + 1000)`, 0},
-		{`5 ** (3 * 2) + 21`, 15646},
```

Comment:
@saveriomiroddi I want to keep these combined arithmetic operations test cases. Can you create another test function for these kind of tests?

Reply:
:+1: Restored!

Reply:
Nice 😎